content/fr/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: OpenTelemetry
+description: >-
+  Une télémétrie de qualité, universelle et portable pour une observabilité efficace
+outputs:
+  - HTML
+  - REDIRECTS # Include this `content/fr` ONLY
+developer_note:
+  La macro "blocks/cover" définie ci-dessous permet d'utiliser comme image de fond tout visuel contenant "background" dans son nom. 
+show_banner: true
+default_lang_commit:
```

Comment:
https://opentelemetry.io/docs/contributing/localization/#the-default_lang_commit-front-matter-field

```suggestion
default_lang_commit: 90dcedd7f692a505eb86145734b1e323ec988820
```